### ISSUE TITLE
runwda fails on iOS 13.7

### ISSUE THREAD

**Issue by nanoscopic** [danielpaulus/go-ios#68]
```
user@Mac-Monterey ios_remote_provider % ./bin/go-ios info | python -m json.tool | grep "\"Product"
    "ProductType": "iPhone9,1", ( iPhone 7 )
    "ProductVersion": "13.7",
user@Mac-Monterey ios_remote_provider % ./bin/go-ios runwda --bundleid com.dryark.CFAgent.xctrunner --testrunnerbundleid com.dryark.CFAgent.xctrunner --xctestconfig CFAgent.xctest
{"bundleid":"com.dryark.CFAgent.xctrunner","level":"info","msg":"Running wda","testbundleid":"com.dryark.CFAgent.xctrunner","time":"2021-10-07T15:44:08-07:00","xctestconfig":"CFAgent.xctest"}
{"error":"_IDE_initiateControlSessionWithCapabilities got wrong returnvalue: [{%!s(uint64=1) DTXMessage map[NSLocalizedDescription:Unable to invoke -[\u003cXCIDESession: 0x119f2ee90\u003e (socket 5) created Thursday, October 7, 2021 at 3:44:08 PM Pacific Daylight Time _IDE_initiateControlSessionWithCapabilities:] - it does not respond to the selector]}]","level":"fatal","msg":"Failed running WDA","time":"2021-10-07T15:44:10-07:00"}
```

It also fails on the following, in I believe the same way:

- iPhone SE 2nd edition iOS 13.5.1
- iPhone 8 iOS 12.4
- iPhone 8 plus iOS 12.3.2
- iPhone 11 iOS 13.7

**Comment by danielpaulus** [comment]
should be resolved with #87

**Comment by mankoffs** [comment]
Hello @danielpaulus! 

I faced the same problem as described above on the latest version of the go-ios (v1.0.69)

`testuser$ ./bin/go-ios info | python -m json.tool | grep "\"Product"
    "ProductName": "iPhone OS",
    "ProductType": "iPhone11,8",
    "ProductVersion": "13.7",
    "ProductionSOC": true,`



`{"error":"_IDE_initiateControlSessionWithCapabilities got wrong returnvalue: [{%!s(uint64=1) DTXMessage map[NSLocalizedDescription:Unable to invoke -[\u003cXCIDESession: 0x15534a150\u003e (socket 7) created Thursday, 19 May 2022, 15:19:46 Eastern European Summer Time _IDE_initiateControlSessionWithCapabilities:] - it does not respond to the selector]}]","level":"fatal","msg":"Failed running WDA","time":"2022-05-19T15:19:47+03:00"`
